### ISSUE TITLE
Ability to disable warning "no department given for MethodLength. Run `rubocop -a --only Migration/DepartmentName` to fix."

### ISSUE THREAD

**Issue by morr** [rubocop-hq/rubocop#7717]
It is very hard for my little brain to remember namespaces for each rubocop rule I disable.

It is much easier to write
```ruby
  def parse value # rubocop:disable MethodLength
```

rather than
```ruby
  def parse value # rubocop:disable Metrics/MethodLength
```

And it is so annoying to manually run in console `rubocop -a --only Migration/DepartmentName`.
Please add ability to disable the nasty warning.

**Comment by bbatsov** [comment]
RuboCop no longer supports ambiguous cop names (those without a department). As the namespace is actually a part of the cop name, this can't be changed.
